app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -1,80 +1,70 @@
-*,
-*:after,
-*:before {
-  box-sizing: border-box;
+.status {
+  margin: 20px 0;
+  clear: both;
 }
 
-.steps-container {
-  overflow: hidden;
-  margin: 0px;
-  padding: 0px;
-  white-space: nowrap;
-  border-left: 2px solid;
-  border-right: 2px solid;
-  width: 100%;
-  counter-reset: steps;
+.status__heading {
+  line-height: 25px;
+  margin: 0;
+  margin-left: 10px;
+  float: left;
+  margin-bottom: 5px;
+  font-weight: normal;
 }
-.steps {
-  position: relative;
-  display: inline-block;
-  left: -28px; /* -2px default + 26px offset to hide skewed area on the left side of first element*/
-  height: 50px;
-  line-height: 50px;
-  margin-left: 0px;
-  margin-right: 0px;
-  counter-increment: steps;
-  cursor: pointer;
-  transition: background 1s;
-  min-height: 30px;
-  width: 100px;
+
+.status__body {
+  clear: both;
+  margin-left: 35px;
+  max-width: 600px;
 }
 
-.steps:after,
-.steps:before {
-  position: absolute;
-  content: '';
-  left: 0px;
-  height: 50%;
-  width: 100%;
-  border-top: 2px solid;
-  border-bottom: 2px solid;
-  border-left: 3px solid; /* thicker border as skew makes them look thin */
-  border-right: 3px solid;
-  background: rgba(255, 255, 255, 0.15);
+.status__icon {
+  float: left;
+  padding: 5px;
+  border-radius: 50px;
+  overflow: hidden;
 }
 
-.steps:before {
-  transform: skew(45deg);
-  top: 0px;
-  border-bottom: none;
-  transform-origin: top left;
+.status__actions {
+  margin-left: 35px;
+  margin-top: 20px;
 }
 
-.steps:after {
-  transform: skew(-45deg);
-  bottom: 0px;
-  border-top: none;
-  transform-origin: bottom left;
+.status__actions button {
+  margin-right: 50px;
 }
 
-.steps span{
+.status__icon:after {
+  width: 15px;
```

Comment:
Properties should be ordered background-position, background-repeat, background-size, content, display, height, width